app/src/main/kotlin/edu/stanford/bdh/engagehf/messages/MessagesAction.kt

```diff
@@ -6,6 +6,7 @@ sealed class MessagesAction {
     data object MeasurementsAction : MessagesAction()
     data class QuestionnaireAction(val questionnaireId: String) : MessagesAction()
     data object HealthSummaryAction : MessagesAction()
+    data object NoAction : MessagesAction()
```

Comment:
Minor: I am more in favor of nullable instead of introducing a new type that will always stay in memory for nothing 😄